README.md

```diff
@@ -4,12 +4,22 @@ Acquia Drupal Recommended Project
 This is a project template providing a great out-of-the-box experience for new Drupal 9 projects hosted on Acquia. It is based on the [Drupal Recommended Project](https://github.com/drupal/recommended-project/tree/9.0.x) and similar to the [Acquia Drupal Minimal Project](https://github.com/acquia/drupal-minimal-project), with the principal difference being the addition of several modules and packages that provide the best possible out-of-the-box experience for Acquia customers.
 
 This project includes the following packages and configuration:
-* Drupal core
-* [Acquia CMS](https://github.com/acquia/acquia_cms) (Drupal distribution)
+* [Drupal core](https://www.drupal.org/project/drupal)
+* [Drupal core scaffold](https://www.drupal.org/docs/develop/using-composer/using-drupals-composer-scaffold)
+* [Acquia CMS](https://github.com/acquia/acquia-cms-starterkit) (Starter kit)
 * [Drush](https://github.com/drush-ops/drush) (Drupal CLI and development tool)
-* [Acquia Connector](https://drupal.org/project/acquia_connector) (Management and reporting tool for Acquia Cloud sites)
-* [Acquia Site Studio](https://www.acquia.com/products/drupal-cloud/site-studio) (Low-code visual page builder)
 * [Asset Packagist](https://asset-packagist.org/) repository, package, and configuration
+* [Acquia environment detection](https://github.com/acquia/drupal-environment-detector)
+* Acquia platform memcache settings
```

Comment:
Any reason we can't link this one too?
https://github.com/acquia/memcache-settings

Reply:
Thanks @balsama. I've made the changes. :)